src/dump.c

```diff
@@ -2603,10 +2624,12 @@ static void jl_insert_backedges(jl_array_t *edges, jl_array_t *ext_targets)
                 ptrhash_remove(&new_code_instance_validate, codeinst);  // should be left invalid
                 codeinst = jl_atomic_load_relaxed(&codeinst->next);
             }
+            invalidate_backedges(&remove_code_instance_from_validation, caller, world, "insert_backedges");
```

Comment:
This is the line we didn't seem to have before, and I'm surprised. Am I missing something?